### PR TITLE
Added unit tests for the resend_welcome_email command.

### PR DESCRIPTION
**Pull request by ShifaSZ**
Both normal command and exception tests are included.

### CODE REVIEW

File: seqr/management/commands/resend_welcome_email_tests.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+from django.core.management.base import CommandError
+
+TEST_USER_EMAIL = 'test_user@test.com'
+REFERRER_EMAIL = 'test_user_no_staff@test.com'
+
+
+class ResendWelcomeEmailTest(TestCase):
+    @mock.patch('seqr.utils.communication_utils.send_welcome_email')
+    @mock.patch('django.contrib.auth.models.User')
```
Comment (hanars) [inline seqr/management/commands/resend_welcome_email_tests.py:15]: Never mock django models. Just make your test work with the fixture data

Comment (ShifaSZ) [reply seqr/management/commands/resend_welcome_email_tests.py]: Updated

File: seqr/management/commands/resend_welcome_email_tests.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+
+class ResendWelcomeEmailTest(TestCase):
+    fixtures = ['users']
+
+    @mock.patch('seqr.utils.communication_utils.send_welcome_email')
+    def test_command(self, mock_send_welcome_email):
+        call_command('resend_welcome_email', '--email-address={}'.format(TEST_USER_EMAIL),
+                     '--referrer={}'.format(REFERRER_EMAIL))
+        user = User.objects.get(email__iexact=TEST_USER_EMAIL)
```
Comment (hanars) [inline seqr/management/commands/resend_welcome_email_tests.py:22]: `iexact` is not necessary and makes the code a bit less readable

Comment (ShifaSZ) [reply seqr/management/commands/resend_welcome_email_tests.py:22]: I removed them.

**Pull request merged**